plugin-server/src/cdp/templates/_destinations/linear/linear.template.ts

```diff
@@ -2,7 +2,7 @@ import { HogFunctionTemplate } from '~/cdp/types'
 
 export const template: HogFunctionTemplate = {
     status: 'stable',
-    free: false,
+    free: true,
```

Comment:
Can we _not_ do this yet? We will be releasing the new pricing model which essentially removes all "free" concepts and makes everything fall under the usage based pricing.

internal destinations (like alerts) will generally be free and not included in this quota.

Reply:
Totally fair. We have a couple of customers that requested this feature that don't have the data pipelines add-on so I was doing it for them. Is there a way to give them access to particular destinations? If the new pricing is imminent I can ask if they're happy to wait

Reply:
Perfect, will set the expectation with them. Thanks 👌